pset-functions.js

```diff
@@ -114,9 +141,16 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+function isPalindrome(word) {
+  const originalStr = word.toLowerCase()
+  const reverseStr = originalStr.split('').reverse().join('')
+  return reverseStr === originalStr
```

Comment:
Really great use of method chaining and the knowledge that a conditional statement returns a boolean value!